### PR TITLE
Add two new fuzz targets

### PR DESCRIPTION
**Pull request by Navidem**
After analyzing jsonnet-introspector results, I came up with these two new fuzz targets to improve the coverage. Want to see how it works.

### CODE REVIEW

**Review by oliverchang — APPROVED** [review]

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  JsonnetVm* jvm = jsonnet_make();
+  jsonnet_import_callback(jvm, ImportCallback, jvm);
+  int error = 0;
+  char* res =
+      jsonnet_evaluate_snippet_multi(jvm, /*filename=*/"", jsonnet.c_str(), &error);
+
+  std::string json;
+  if (error == 0 && res != nullptr) {
+    json = res;
+  }
```
Comment (DavidKorczynski) [inline projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:44]: These lines do not seem needed?

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  if (res) {
+    jsonnet_realloc(jvm, res, 0);
+  }
+  jsonnet_destroy(jvm);
+  return json;
```
Comment (DavidKorczynski) [inline projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:50]: No need for `return json;` as it's not used

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
+  std::string fuzz_jsonnet(reinterpret_cast<const char*>(data), size);
+  ConvertJsonnetToJson(fuzz_jsonnet);
```
Comment (DavidKorczynski) [inline projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:55]: return value not used

File: projects/jsonnet/convert_jsonnet_fuzzer_stream.cc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  res[0] = 0;
+  return res;
+}
+
+std::string ConvertJsonnetToJson(const std::string& jsonnet) {
+  JsonnetVm* jvm = jsonnet_make();
+  jsonnet_import_callback(jvm, ImportCallback, jvm);
+  int error = 0;
+  char* res =
+      jsonnet_evaluate_snippet_stream(jvm, /*filename=*/"", jsonnet.c_str(), &error);
```
Comment (DavidKorczynski) [inline projects/jsonnet/convert_jsonnet_fuzzer_stream.cc:39]: Does it make sense to merge the two fuzzers as this line seems to be the ony difference? i.e. merge by just calling `jsonnet_evaluate_snippet_stream` and `jsonnet_evaluate_snippet_multi` in the same fuzzer

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc (hunk 1 above)
Comment (Navidem) [reply projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:44]: these lines are coming from initial fuzz target.

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc (hunk 2 above)
Comment (Navidem) [reply projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:50]: same as previous.

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc (hunk 3 above)
Comment (Navidem) [reply projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:55]: same as previous.

File: projects/jsonnet/convert_jsonnet_fuzzer_multi.cc (hunk 1 above)
Comment (DavidKorczynski) [reply projects/jsonnet/convert_jsonnet_fuzzer_multi.cc:44]: the code still doesn't not have any impact -- it converts a `char *` to a `std::string` and returns the string, which is never used. Maybe remove that code from the original then? I checked up and it seems the original fuzz target has not maintained in a while and the used who added does not much activity the last few years. In either case, it won't have much impact so if you would prefer to just merge then I'm more than happy with that.

File: projects/jsonnet/convert_jsonnet_fuzzer_stream.cc (hunk 4 above)
Comment (DavidKorczynski) [reply projects/jsonnet/convert_jsonnet_fuzzer_stream.cc:39]: I would probably prefer what you initially did then -- but either way works!

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**